### PR TITLE
Various tweaks & bug fixes

### PR DESCRIPTION
**Pull request by alanpeabody**
@gblakeman @Justin-Holmes I just addressed some issues I found once we had it working with production data. It might be better to review this one commit by commit, as they are all independent issues.

A lot of the issues are only issues when the data is not getting bootstrapped by redux persist - so any time you visit an artist invite for the first time or you clear localstorage.

### CODE REVIEW

File: src/components/ArtistInviteDashboard/ArtistInviteStatus.js [hunk 1]
```diff
@@ -45,14 +47,14 @@ const getStatusText = (status) => {
...
+const ArtistInviteStatus = ({ status, openedAt, closedAt }) => (
   <StatusHolder className={'status-holder ' + status}>
     <span className="current-status">{getStatusText(status)}</span>
     &nbsp;
     <ArtistInviteCountDown
       status={status}
-      openedAt={(new Date("October 31, 2017 11:13:00")).toString()}
-      closedAt={(new Date("December 1, 2017 11:13:00")).toString()}
+      openedAt={openedAt}
+      closedAt={closedAt}
```
Comment (gblakeman) [inline src/components/ArtistInviteDashboard/ArtistInviteStatus.js]: Nice catch. I stubbed that in for styling and forgot to get back to it. 🙃

File: src/containers/ArtistInviteDashboardContainer.js [hunk 2]
```diff
@@ -208,16 +208,20 @@ class ArtistInviteDashboardContainer extends Component {
...
-          <ArtistInviteCard
-            imgSrc={artistInvite.get('headerImage').getIn(['optimized', 'url'])}
-            title={artistInvite.get('title')}
-            type={artistInvite.get('inviteType')}
-          />
-          <ArtistInviteStatus
-            status="open"
-          />
-        </ArtistInviteHeader>
+        {artistInvite.get('id') &&
```
Comment (gblakeman) [inline src/containers/ArtistInviteDashboardContainer.js]: Would it make more sense to do some placeholder stuff (at least for the header structure)? Trying this on slow (airplane) wifi, and content jumps down the page significantly once everything loads. Not a deal-breaker, for sure, but maybe worth a followup story?

File: src/components/ArtistInviteDashboard/ActivityBar.js [hunk 3]
```diff
@@ -57,55 +57,73 @@ export default class ActivityBar extends Component {
...
-                offsetY={40}
-                domain={{ x: [0, 6] }}
-                tickFormat={(t) => this.capitalize(t)}
-                style={{
-                  axis: {stroke: "translucent"},
-                  tickLabels: {fontSize: 12, fontFamily: typeface.regular, fill: colors.black},
-                }}
-              />
-              <VictoryBar
+        {this.isLoaded() ?
```
Comment (gblakeman) [inline src/components/ArtistInviteDashboard/ActivityBar.js]: Proposed some styling for this loading state in 2cad95a80696f99e2ee98f56a2648034b9b8cf4b – feel free to revert if you’re not a fan.

File: src/containers/ArtistInviteDashboardContainer.js (hunk 2 above)
Comment (alanpeabody) [reply src/containers/ArtistInviteDashboardContainer.js]: Yeah, we can do that for sure.

**Pull request merged**